monarch_mapping_commons.Makefile

```diff
@@ -12,41 +12,51 @@ METADATA_DIR = 					metadata
 RUN = poetry run
 
 
+benchmark:
+	pip install py-spy
+	sudo py-spy record -o flamegraph.svg -- $(SSSOM_TOOLKIT) validate $(MAPPING_DIR)/gene_mappings.sssom.tsv
+
+
 $(MAPPING_DIR)/ $(SCRIPT_DIR)/ $(SRC_DIR)/ $(TMP_DIR)/:
 	mkdir -p $@
 
+
 $(MAPPING_DIR)/mondo.sssom.tsv:
 	mkdir -p $(MAPPING_DIR) $(TMP_DIR)
 	wget http://purl.obolibrary.org/obo/mondo/mappings/mondo.sssom.tsv -O $@
 
+
 $(MAPPING_DIR)/mesh_chebi_biomappings.sssom.tsv:
 	mkdir -p $(MAPPING_DIR) $(TMP_DIR)
 	wget https://raw.githubusercontent.com/biopragmatics/biomappings/master/docs/_data/sssom/biomappings.sssom.tsv -O $(TMP_DIR)/biomappings.sssom.tsv
 	$(RUN) python3 $(SCRIPT_DIR)/process_biomappings.py --input $(TMP_DIR)/biomappings.sssom.tsv --output $(TMP_DIR)/mesh_chebi_biomappings.sssom.tsv
 	$(RUN) sssom parse $(TMP_DIR)/mesh_chebi_biomappings.sssom.tsv -m $(METADATA_DIR)/mesh_chebi_biomappings.sssom.yml --prefix-map-mode merged -o $@
 
+
 $(MAPPING_DIR)/gene_mappings.sssom.tsv:
+ifndef GH_ACTION
 	mkdir -p $(MAPPING_DIR) $(TMP_DIR)
-	wget http://data.monarchinitiative.org/monarch-gene-mapping/latest/gene_mappings.sssom.tsv -O $(TMP_DIR)/gene_mappings.sssom.tsv
+	$(RUN) gene-mapping generate --download --preprocess-uniprot --output-dir $(TMP_DIR)
 	$(RUN) sssom parse $(TMP_DIR)/gene_mappings.sssom.tsv -m $(METADATA_DIR)/gene_mappings.sssom.yml --prefix-map-mode merged -o $@
+else ifeq ($(GH_ACTION), true)
+	echo "Gene Mappings target is unavailable in GitHub actions."
+endif
```

Comment:
How will we make sure that, when you make a PR that changes anything related to gene-mappings, the resulting sssom file is still correct?

Reply:
Maybe we can add, at least, an SOP for this case. I would encode the SOP as a "pull request template", with a checkbox saying something like

- I have ensured that `make mapping` still works correctly

Here is how we do this, for example, in ROBOT: https://github.com/ontodev/robot/blob/master/.github/pull_request_template.md